pkg/event/target/mqtt.go

```diff
@@ -21,15 +21,19 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"net/url"
 	"path/filepath"
+	"strconv"
 	"time"
 
-	"github.com/eclipse/paho.mqtt.golang"
+	"github.com/Praveenrajmani/paho.mqtt.golang"
```

Comment:
I believe you have already sent the fix to `paho.mqtt.golang`. How about keeping this under `pkg/event/target/paho.mqtt.golang` till the PR merged in upstream?

Reply:
Will do :+1:

Reply:
Done